lib/fastly_nsq/rake_task.rb

```diff
@@ -15,68 +16,50 @@ def initialize(*args, &task_block)
             yield(*[self, task_args].slice(0, task_block.arity))
           end
 
-          if task_args[:channel]
-            @channel = task_args[:channel]
-          end
+          @channel  ||= require_arg :channel, task_args 
+          @topics   ||= require_arg :topics, task_args
+          @listener ||= task_args[:listener]
```

Comment:
> This moves the addition of new topics (ATM) to listen to into a cookbook change

@alieander I'm missing something, but I don't see that happening. I've got a Proof of Concept / Demo put together in one of our apps, but it doesn't require cookbook changes, as far as I can tell. Elaborate if you can on what you think I'm missing?